helpers/test-data-generator.js

```diff
@@ -54,7 +54,7 @@ export async function executeSignedRoleAssignment({ colony, functionName, taskId
 export async function setupAssignedTask({ colonyNetwork, colony, dueDate, domain = 1, skill = 0, evaluator, worker }) {
   const accounts = await web3GetAccounts();
   const manager = accounts[0];
-  evaluator = evaluator || accounts[1]; // eslint-disable-line no-param-reassign
```

Comment:
We default the evaluator to `accounts[1]` in a few more places in the test-data-generator which need changing

Reply:
Updated